templates/dispatch.yaml

```
@@ -0,0 +1,30 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: dispatch
+  labels:
+    kubeaddons.mesosphere.io/name: dispatch
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0"
```

It seems like most templates have `<appversion>-1` for revision but not 100% sure if the `-1` is required. Maybe @shaneutt can confirm what's expected here?

---

Shane's out for the weekend. How do we test that this works in the frontend without the -1?

---

I tested this on my own konvoy cluster and everything installs. It does not show in the ops portal dashboard, but my understanding is it is not supposed to yet, according to @natmegs:

> if ingress is set up it should be reachable on the cluster but the links have not yet been added to kommander/konvoy UIs

---

The part I'm wondering about is how the revision annotation could affect catalog API

---

The catalog will not support parsing this repo right now if this annotation isn't there, and in the correct format. It's a hack, we have tech debt JIRA stuff to clean this up but is necessary for now.

This should be changed to the following:

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0-1"
```